packages/flutter/lib/src/widgets/scroll_activity.dart

```diff
@@ -336,8 +435,11 @@ class DrivenScrollActivity extends ScrollActivity {
   Completer<Null> _completer;
   AnimationController _controller;
 
+  /// A [Future] that completes when the activity stops.
```

Comment:
Add a discussion that this does not guarantee that the animation will have reached `to` or that the given `duration` will have elapsed.